assets/Changelog.md

```diff
@@ -120,10 +141,6 @@ Add additional extensions to USMap dumper - Atenfyr; Archengius
 
 Fix bug in USMap dumper with enums with 256 entries - Atenfyr
 
-### Live View
-
-### UHT Dumper
-
```

Comment:
I wanted to keep empty sections so that it's clear exactly where to put things for the future.
Without them, people will keep re-creating and re-ordering them randomly leaving readers with no consistency for future changelogs.

Reply:
Fine